docs/repo-docs/guides/tools/typescript.mdx

```diff
@@ -292,15 +294,15 @@ In [Compiled packages](https://turbo.build/repo/docs/core-concepts/internal-pack
 </Tab>
 
   <Tab value="Source code">
+
 ```tsx title="./packages/ui/button.tsx"
-import { MY_STRING } from "#utils.js" // Uses .js extension // [!code highlight]
+import { MY_STRING } from '#utils.js'; // Uses .js extension // [!code highlight]
```

Comment:
Is something wrong with our formatter? 🤔

Reply:
no idea how/what happened there.  fixed in 048e3f53c0a49db9e98ea68bb90df2ecf4bba084

Reply:
Look like this issues is still here - that fix looks like it's in another file!

Reply:
oh geeze: I messed it up. and that right there is why I always attach the commit where I intended to fix something.  how's this: 84591252fc6aec6a7113914d1d0d8a38e7bf0a6f

Reply:
what in the world is going on.  something is messed up with my checkout of turborepo. fixing.

Reply:
5ded22cce281d503c5a6e03957bb2223c8bf7c3e

I think there's something going on with the pre-commit hook.  it was changing things out from under me - very confusing.